File: src/main/scala/br/com/fabioaraujo/calcula/produto/service/HoraTrabalhadaService.scala
package br.com.fabioaraujo.calcula.produto.service

import br.com.fabioaraujo.calcula.produto.dao.{HoraTrabalhadaRepository, ProdutoRepository}
import br.com.fabioaraujo.calcula.produto.dto.{HoraTrabalhada, Produto}
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class HoraTrabalhadaService(@Autowired val horaTrabalhadaRepository: HoraTrabalhadaRepository) {

  def getHora(): HoraTrabalhada = {
    horaTrabalhadaRepository.findOne(1L)
  }

  def criar(horaTrabalhada: HoraTrabalhada): Long = {
    horaTrabalhada.id = 1L
    horaTrabalhadaRepository.save(horaTrabalhada)
    horaTrabalhada.id
  }
}

File: src/main/java/br/com/fabioaraujo/calcula/produto/service/HoraTrabalhadaService.java
package br.com.fabioaraujo.calcula.produto.service;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import br.com.fabioaraujo.calcula.produto.dao.HoraTrabalhadaRepository;
import br.com.fabioaraujo.calcula.produto.dto.HoraTrabalhada;

@Service
public class HoraTrabalhadaService {

	@Autowired 
	private HoraTrabalhadaRepository horaTrabalhadaRepository;

	public HoraTrabalhada getHora() {
		return horaTrabalhadaRepository.findById(1L).orElse(null);
	}

	public Long criar(HoraTrabalhada horaTrabalhada) {
		horaTrabalhada.setId(1L);
		horaTrabalhadaRepository.save(horaTrabalhada);
		return horaTrabalhada.getId();
	}

}